docs/services/ml/upload-training-script.md

```diff
@@ -0,0 +1,611 @@
+---
+title: "Train a Model with a Custom Python Training Script"
+linkTitle: "Use a Custom Training Script"
+weight: 90
+type: "docs"
+tags: ["data management", "ml", "model training"]
+description: "How to write and upload a custom machine learning model training script to the Viam registry and use it to submit training jobs."
+no_service: true
+# SME: Tahiya S.
+---
+
+You can upload a custom training script to the [Viam registry](https://app.viam.com/registry/) to train your ML model.
```

Comment:
This is missing more of an introduction. I think we need to explain why you would be using it instead of the regular training flow.

Reply:
@tahiyasalam Would you be able to help provide any more context here-- why have customers been asking for this feature? Why would a regular user use it?

Reply:
Here's a tidbit from [the scope](https://docs.google.com/document/d/1nuqYF2tTx5fcYb7UaJTfKRJmGlA1jim3gF-dXdsw7YU/edit#heading=h.tcicyojyqi6c): Machine learning training is currently limited to classification and object detection, where the trained model is output as a TFLite model. A flexible training interface will allow more generic training by allowing users to upload any machine learning training code for their use-case. The model output type can be PyTorch, Tensorflow, TFLite, ONNX, or any other model artifacts.

Reply:
added